app/src/main/java/com/benefit/ConversationActivity.java

```diff
@@ -127,4 +128,55 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
         userService.handleOnSignInResult(requestCode, resultCode, data, this);
     }
+
+    private void setHeaderListeners() {
+        findViewById(R.id.give_icon).setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                startGiveActivity();
+            }
+        });
+
+        findViewById(R.id.give_icon).setOnClickListener(new View.OnClickListener() {
```

Comment:
You forgot to change the buttons. For example, should be user_icon  here

Reply:
fixed